app/cdap/components/hydrator/components/TopPanel/ActionButtons/index.tsx

```diff
@@ -133,7 +133,9 @@ export const ActionButtons = ({
     }
   };
 
-  const configureButtonRef = useRef(null);
+  const configureButtonAnchorEl = previewMode
```

Comment:
the anchor element is just a string? if this is the how you want to do it we should probably rename this variable since its no longer a reference to an anchor el but a selector or something?

Reply:
Yeah I think its probably fine as the selector - I was more saying change the variable name to include selector or something like `anchorElSelector` or something like that - I think this is fine either way though because that component specifically allows both string and element.